### ISSUE TITLE
Feature request: New interface & new functions

### ISSUE THREAD

**Issue by r57zone** [qTox/qTox#4497]
![](https://user-images.githubusercontent.com/9499881/27600548-6169b6a8-5b7d-11e7-8570-cf39d3c4c14b.png)
If the interface is slightly improved, it will become more convenient and more people will be able to go to it. For example, as on Telegram. Calm for eye color and minimal interface. Can generally make a clone and then it will be perfect.

Also [duplicate feature](https://github.com/TokTok/c-toxcore/issues/568) request on toxcore:

Conferences like on old p2p Skype.

1. Url to join - qtox://ID_AUTHOR?Join=GROUP_USER_ID (connect to user and add to group)
2. Save conference group after exit qTox (sync between users, until the author removes it.).
3. Save local history on clients and load history messages after connect to conference (sync between users).
4. Commands (like on skype) and permission (moderator - delete messages, bans, or etc)
5. Maybe delete all messages after leave conference.

All this is important to become a user-friendly client.

Thanks.

**Comment by SergeyDjam** [comment]
1. toxcore kernel doesn't support constant groups
2-4 tox doesn't support is
if you leave group all messages kill

**Comment by r57zone** [comment]
@SergeyDjam What about the new interface? This is important for user-friendly. Maybe can gradually change it?

**Comment by Diadlo** [comment]
@SergeyDjam TokTok/c-toxcore#217

**Comment by SergeyDjam** [comment]
@Diadlo, the patient is rather dead than living

**Comment by SergeyDjam** [comment]
@r57zone, the new interface isn't so important as a new functionality

**Comment by r57zone** [comment]
@SergeyDjam The interface is very important! Because it is used every day and it should be pleasant to the eyes. Telegram does not differ much from Tox in the interface, but it is more pleasant and convenient to read and write. It is necessary to change very little so that everyone is happy. 

Need to change quite a bit (colors, border padding, some elements to enlarge, some to move, some to transfer), the interfaces are similar.

A beautiful and user-friendly interface will give more users of qTox.

**Comment by SergeyDjam** [comment]
@r57zone, The interface important piece, but to whom it is necessary when there is no working things?

**Comment by r57zone** [comment]
@SergeyDjam Сan do it in parallel. While someone is making a kernel, can update the interface.

**Comment by ghost** [comment]
Frankly, I much prefer the current qTox interface to the example in the original post (if that was intended as an example).

**Comment by SergeyDjam** [comment]
@OliverGITReport, yes i too, but preview images, video, markdown, code view and stikers are very necessary in a chat window

**Comment by r57zone** [comment]
The current interface is very engineering, it's probably good for engineers, but not so good for everyone, maybe you just got used to it? I think it's outdated, chatView exactly. White background and elements need change, in Telegram is faster distinguishable elements and a nice blue background. A white background burns the eyes.

Let's just move in this direction? To create a product for everyone. Pleasant, beautiful, functional and safe.

**Comment by zetok** [comment]
@r57zone NB: for the most efficient handling of things, it's best to make separate issues for separate things – e.g. it wouldn't be good to have discussion about UI improvement derail because of other, not related to UI points.

Of course one can make a meta-issue about the general direction in which things are supposed to go, but in that case it should just link to other issues that are about separate things.

1, 4, 5 – separate issues
2, 3 – new groupchats are supposed to handle that, there are open issues (be it in qTox or toxcore repos)

---

Since this issue is about UI, lets just focus on it, with other points being addressed in other issues.

---

> White background and elements need change, in Telegram is faster distinguishable elements and a nice blue background.

I imagine that better background wouldn't be just a change of color? Could you provide a better background? A .svg one, so that it would work well on high-dpi displays.

Does the background change conflict with #1639 ? If not, how would it be done?

Related: #1310

**Comment by Diadlo** [comment]
Conflict with #4135?

**Comment by Hasshu** [comment]
> A beautiful and user-friendly interface will give more users of qTox.

@r57zone Not to sound rude, but the flat abomination you showed in the OP is anything but this.

**Comment by r57zone** [comment]
@Hasshu This is not rude, i had in mind to move on and make the interface better than now. Do not stand still. I like Tox and I want Tox to become a messenger for everyone.

**Comment by tr37ion** [comment]
@r57zone Maybe, you can start with branching the code and then implement your UI. Then we will see.

To me the proposed UI is not of my taste, but that is a personal thing and has nothing to say at all. I agree, Tox needs a more pleasant/modern UI. I sometimes have difficulties to explain ordinary users how to use it or even how to use/set it up.

Some UI examples of similar application UIs

 ![Mattermost](https://about.mattermost.com/wp-content/uploads/2016/08/group.png)
 ![Discord](http://i.imgur.com/x52XcAY.png)
 ![Matrix](https://matrix.org/blog/wp-content/uploads/2017/03/iMac-V2-low-res.png)
 
There seems to be a tendency to move categories to the left and chat room related content to the right.
